ci-operator/step-registry/cucushift/installer/check/vsphere/template/cucushift-installer-check-vsphere-template-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set +x
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+installer_dir=/tmp/installer
+echo "$(date -u --rfc-3339=seconds) - Copying config from shared dir..."
+
+mkdir -p "${installer_dir}"
+pushd ${installer_dir}
+cp -t "${installer_dir}" \
+    "${SHARED_DIR}/install-config.yaml"
+
+template_config=$(cat install-config.yaml | grep template |awk -F ' ' '{print $2}')
```

Comment:
I think we don't need to copy install-config.yaml file from SHARED_DIR, but directly read "${SHARED_DIR}/install-config.yaml". 

And suggest to use yq-go instead to get field value from install-config, then check that template under each failureDomain matches with expected template.
`template_config=yq-go r "${SHARED_DIR}/install-config.yaml" 'platform.vsphere.failureDomains[*].topology.template'`

Reply:
good catch. let me update